libcxx/utils/libcxx/test/features.py

```diff
@@ -183,25 +183,26 @@ def _getAndroidDeviceApi(cfg):
         actions=[AddLinkFlag("-latomic")],
     ),
     Feature(
-        name="non-lockfree-atomics",
+        name="has-64-bit-atomics",
```

Comment:
I think this name is both an improvement (because it calls out the size being the factor here), but also gets worse because locking atomics are still atomic. I wouldn't understand the meaning of this feature check if I were just reading it in the test.

Reply:
I think the realization here is that the previous name `non-lockfree-atomics` was entirely wrong. In theory, if a system supported lockfree atomic operations for `sizeof(int)*100`, this feature would have a downright incorrect name.

Reply:
What does it mean to have `has-64-bit-atomics`? 

I guess I still don't understand the fundamental underlying property we're trying to observe.